airbyte-db/src/main/java/io/airbyte/db/Databases.java

```diff
@@ -109,4 +116,23 @@ private static BasicDataSource createBasicDataSource(final String username,
     return connectionPool;
   }
 
+  public static String getPostgresJdbcUrl(JsonNode config) {
```

Comment:
see my comment below, but i don't think using `JsonNode config` here is very safe. It's hard to know if this is the config for pg source, pg destination, or something else.